pyvisa-py/gpib.py

```diff
@@ -189,6 +189,49 @@ def clear(self):
         except:
             return 0, StatusCode.error_system_error
 
+    def gpib_command(self, commandByte):
+        """Write GPIB command byte on the bus.
+
+        Corresponds to viGpibCommand function of the VISA library.
+        See: https://linux-gpib.sourceforge.io/doc_html/gpib-protocol.html#REFERENCE-COMMAND-BYTES
+
+        :param commandByte: command byte to send
+        :type commandByte: int, must be [0 255]
+        :return: return value of the library call
+        :rtype: :class:`pyvisa.constants.StatusCode`
+        """
+
+        if 0 <= commandByte <= 255:
+            data = bytes([commandByte])
+        else:
+            return StatusCode.error_nonsupported_operation
+
+        try:
+            self.controller.command(data)
+            return StatusCode.success
+
+        except KeyError:
```

Comment:
It is not obvious to me that this is the right error to catch (looking at https://github.com/tivek/gpib_ctypes/blob/master/gpib_ctypes/gpib/gpib.py#L235).